services/webnn/ort/graph_builder_ort.cc

```diff
@@ -2277,9 +2275,10 @@ GraphBuilderOrt::AddLstmOperation(const LstmType& lstm) {
                                           recurrent_weight_shape[1]}));
   }
   if (lstm.layout == mojom::LstmWeightLayout::kIfgo) {
-    ASSIGN_OR_RETURN(weight, ConvertRnnGateLayout(weight, 4));
-    ASSIGN_OR_RETURN(recurrent_weight,
-                     ConvertRnnGateLayout(recurrent_weight, 4));
+    ASSIGN_OR_RETURN(weight,
+                     TransposeRnnWeightOrBiasLayout(weight, {0, 3, 1, 2}));
```

Comment:
Do the same for `{0, 3, 1, 2}`, e.g.
```suggestion
                     TransposeRnnWeightOrBiasLayout(weight, kIfgoToIofgPermutation));
```

Reply:
Done.